Mathlib/RingTheory/DedekindDomain/Ideal.lean

```diff
@@ -795,6 +827,38 @@ theorem Ideal.exist_integer_multiples_notMem {J : Ideal A} (hJ : J ≠ ⊤) {ι
 @[deprecated (since := "2025-05-23")]
 alias Ideal.exist_integer_multiples_not_mem := Ideal.exist_integer_multiples_notMem
 
+lemma Ideal.mul_iInf (I : Ideal A) {ι : Type*} [Nonempty ι] (J : ι → Ideal A) :
```

Comment:
Can you check that we have the corresponding lemmas for `sup`? I see for example [Ideal.mul_sup](https://leanprover-community.github.io/mathlib4_docs/Mathlib/RingTheory/Ideal/Operations.html#Ideal.mul_sup) and [Ideal.sup_mul](https://leanprover-community.github.io/mathlib4_docs/Mathlib/RingTheory/Ideal/Operations.html#Ideal.sup_mul) but nothing else.